data/Act.csv

```diff
@@ -572,7 +612,7 @@ A00817,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing
 A00818,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing secondary schools and were stored in an office after the schools were moved to the countryside,SS00006,134,2,unknown work by: P0999 G02 about P0232,,2018-10-24,MG,2018-10-24,MG
 A00819,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing secondary schools and were stored in an office after the schools were moved to the countryside,SS00006,134,2,unknown work by: P0999 G02 about P0651,,2018-10-24,MG,2018-10-24,MG
 A00820,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing secondary schools and were stored in an office after the schools were moved to the countryside,SS00006,134,2,unknown work by: P0999 G02 about P0652,,2018-10-24,MG,2018-10-24,MG
-A00821,P0120,AT23,PS00490,,1966..197x,1966..197x,oral transmission,SS00030,30; 152-153; 220; 251,2,,,2018-10-24,MG,2018-10-24,MG
```

Comment:
removed on purpose?

Reply:
no, didn't want to remove it...

Reply:
in which case we should put the page ranges back in `30; 152-153; 220; 251`